roles/matrix-nginx-proxy/defaults/main.yml

```diff
@@ -227,9 +227,18 @@ matrix_nginx_proxy_proxy_matrix_federation_api_ssl_certificate_key: "{{ matrix_s
 # The tmpfs at /tmp needs to be large enough to handle multiple concurrent file uploads.
 matrix_nginx_proxy_tmp_directory_size_mb: "{{ (matrix_nginx_proxy_proxy_matrix_federation_api_client_max_body_size_mb | int) * 50 }}"
 
+# A list of strings containing additional configuration blocks to add to the nginx server configuration (nginx.conf).
+# for big matrixservers to enlarge the number of open files to prevent timeouts
+# matrix_nginx_proxy_proxy_additional_server_configuration_blocks:
+#  - 'worker_rlimit_nofile 30000;'
+matrix_nginx_proxy_proxy_additional_server_configuration_blocks: []
```

Comment:
Should we remove the `_server_` part from this variable's name?

It kind of implies this ends up in a `server {}` block? Or that this is for defining new `server {}` blocks?
Maybe:

- `matrix_nginx_proxy_proxy_additional_configuration_blocks`
- `matrix_nginx_proxy_proxy_event_additional_configuration_blocks`

Reply:
Ok i change it. 
Is it possible to do something like this:
```
matrix_nginx_proxy_proxy_additional_configuration_blocks:
  - http { server { listen 80; server_name foo; rooot /foo} }
  ```
to add additional websites to nginx? I am not sure whether it can handle two http blocks. Otherwise i would add this feature in line 59 to nginx.conf.j2

Reply:
Having not tested it, not sure if one can add a 2nd `http` block.

Reply:
2nd http block does not work, but `matrix_nginx_proxy_proxy_http_additional_server_configuration_blocks:` allready does this
```
matrix_nginx_proxy_proxy_http_additional_server_configuration_blocks::
  - http { server { listen 8080; server_name foo; root /foo} }
```
and a port  above 1024